runtime/compiler/aarch64/runtime/PicBuilder.spp

```diff
@@ -57,6 +57,7 @@
 	.globl	FUNC_LABEL(_nativeStaticHelper)
 	.globl	FUNC_LABEL(_interfaceCompleteSlot2)
 	.globl	FUNC_LABEL(_interfaceSlotsUnavailable)
+	.globl  FUNC_LABEL(_patchGCRHelper)
```

Comment:
Please use a tab character between `.globl` and `FUNC_LABEL`.

Reply:
Changed it to a tab.